index.js

```diff
@@ -1,80 +1,81 @@
 module.exports = {
-  "extends": [
+  extends: [
     'plugin:@typescript-eslint/recommended',
     'plugin:@typescript-eslint/recommended-requiring-type-checking',
     'prettier/@typescript-eslint',
-    'plugin:prettier/recommended'
+    'plugin:prettier/recommended',
   ],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  },
-  "plugins": [
-    "import",
-    "@typescript-eslint"
-  ],
-  "settings": {
-    "import/parsers": {
-      "@typescript-eslint/parser": [".ts"]
+  parser: '@typescript-eslint/parser',
+  plugins: ['import', '@typescript-eslint'],
+  settings: {
+    'import/parsers': {
+      '@typescript-eslint/parser': ['.ts', '.tsx'],
```

Comment:
Wondering if we should just implement the `@blockstack/eslint-config/react` file now, and only include `.tsx` in that one?

Reply:
🤷‍♂ 

I know we might want to do this at some point, for a first run don't see a problem. A node project will have no .tsx thus it'll return/lint any files